enterprise/backend/test/metabase_enterprise/airgap_test.clj

```diff
@@ -0,0 +1,36 @@
+(ns metabase-enterprise.airgap-test
+  (:require    [clojure.java.io :as io]
+               [clojure.string :as str]
+               [clojure.test :refer :all]
+               [metabase-enterprise.airgap :as airgap]
+               [metabase.public-settings.premium-features :as premium-features]))
+
+(defn- test-fake-features [& {:keys [token-fn pubk-fn]}]
```

Comment:
for changing settings we usually use `mt/with-temporary-setting-values` or similar.